setupdocs.md

```diff
@@ -0,0 +1,182 @@
+
+# Car-Bon System: Technical Setup Guide
+
+## Prerequisites
+
+Before beginning the installation, ensure you have:
+
+-   Administrator access to the project repository of github
+-   Windows operating system
+-   Basic understanding of command-line interfaces
+
+## Version Control Setup
+
+1.  Install Git
+    -   Download and install from the official website
+    -   Configure with your credentials
+2.  Clone Project Repository
+    
+    bash
+    
+    Copy
+    
+    `git clone https://github.com/gunnnu/Car-Bon_UI.git`
+    
+    **Note:** Repository access requires admin invitation
+
+## Windows Subsystem for Linux (WSL) Configuration
+
+1.  Open PowerShell as Administrator
+    
+    powershell
+    
+    Copy
+    
+    `wsl --install`

```

Comment:
As mentioned earlier please refrain from using AI tools for writing documentation 
The correct way is 

```powershell
    
    wsl --install
```